bigtable-hbase-1.x-parent/bigtable-hbase-1.x-hadoop/pom.xml

```diff
@@ -147,6 +151,27 @@ limitations under the License.
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>license-maven-plugin</artifactId>
+        <version>2.0.0</version>
+        <executions>
+          <execution>
+            <id>default-cli</id>
+            <phase>generate-resources</phase>
+            <goals>
+              <goal>add-third-party</goal>
+            </goals>
+            <configuration>
+              <excludedScopes>test,provided,system</excludedScopes>
+              <excludedArtifacts>jsr305|metrics-core|commons-logging|hbase-shaded-client|slf4j-api|slf4j-log4j12|log4j|htrace-core|findbugs-annotations|conscrypt-openjdk-uber|jaxb-api</excludedArtifacts>
+              <excludedGroups>org.bouncycastle|org.checkerframework|javax.annotation</excludedGroups>
+              <excludeTransitiveDependencies>true</excludeTransitiveDependencies>
```

Comment:
this should mirror the shade plugin config and simply include the bigtable-hbase-1x-shaded